x-pack/test_serverless/functional/test_suites/common/discover/group1/_discover_histogram.ts

```diff
@@ -67,7 +67,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.common.navigateToApp('discover');
       await PageObjects.discover.waitUntilSearchingHasFinished();
       // this is the number of renderings of the histogram needed when new data is fetched
-      let renderingCountInc = 1;
+      let renderingCountInc = 3;
       const prevRenderingCount = await elasticChart.getVisualizationRenderingCount();
       await queryBar.submitQuery();
```

Comment:
@markov00 does this seem correct to you? I seems like https://github.com/elastic/elastic-charts/pull/2270 made the chart render 2 additional times or so the status claims.

Reply:
I also notice that some time the render-complete status if `false`. Let me check what is going on before continuing this PR

Reply:
checked together and the rendering is triggered twice due to the filter bar gaining focus and collapsing the time picker.

Reply:
Related to https://github.com/elastic/kibana/issues/177055